app/models/container_node.rb

```diff
@@ -115,4 +117,14 @@ def external_logging_path
     index = ".operations.*"
     EXTERNAL_LOGGING_PATH % {:index => index, :query => query}
   end
+
+  def disconnect_inv
+    return if ems_id.nil?
+    _log.info "Disconnecting Node [#{name}] id [#{id}] from EMS [#{ext_management_system.name}]" \
+    "id [#{ext_management_system.id}] "
+    self.old_ems_id = ems_id
+    self.ext_management_system = nil
+    self.deleted_on = Time.now.utc
+    save
+  end
```

Comment:
@zeari yes we need the hardware (it is important that we keep the number of cpus and memory).